src/test/scala/extraction/DeclaredImplicitsTest.scala

```diff
@@ -0,0 +1,112 @@
+package extraction
+
+import framework.SemanticdbTest
+
+class DeclaredImplicitsTest extends SemanticdbTest {
+  checkExtraction(
```

Comment:
Here is an idea - how about we use the case classes as just containers for data. This way, we could update the tests to something like

```scala
  implicit class NormalizedResult(that: Result) {
    // removes location property
    def normalizedImplicits: Set[DeclaredImplicit] = that.implicits.map(_.copy(loc = null))
  }

  checkExtraction(
    "Basic information about declared implicits",
    """
      |package dI
      |object basicInfo {
      | implicit def m(a: String): String = "3"
      |}
    """.trim.stripMargin, { res =>

      res.normalizedImplicits should contain only (
        DeclaredImplicit(
          fqn = "_root_.dI.basicInfo.m(Ljava/lang/String;)Ljava/lang/String;.",
          clazz = "_root_.scala.Predef.String#",
          typee = "(a: String): String",
          kind = "def",
          nargs = "1
        )
      )

      res.funs shouldBe empty
      res.links shouldBe empty
      res.params shouldBe empty
  })
```